examples/flutter_counter/test_driver/screens/app_screen.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+class AppScreen {
+  AppScreen(FlutterDriver flutterDriver) {
+    _driver = flutterDriver;
+  }
+
+  final incrementFloatingButton =
+      find.byValueKey('counterView_increment_floatingActionButton');
+  final decrementFloatingButton =
+      find.byValueKey('counterView_decrement_floatingActionButton');
+  final appBarText = find.text('Counter');
+  final counterState = find.byValueKey('counterState');
+  FlutterDriver _driver;
+
+  /// verify the AppBar text is Counter
+  Future<void> verifyTheAppBarText() async {
+    expect(await _getTextByScreen(appBarText), 'Counter');
+  }
+
+  Future<void> verifyCounterTextIsZero() async {
+    expect(await _getTextByScreen(counterState), '0');
+  }
+
+  Future<void> pressIncrementFloatingActionButtonTwice() async {
+    // tap floating action button
+    await _driver.tap(incrementFloatingButton);
+    expect(await _getTextByScreen(counterState), '1');
+
+    // tap floating action button
+    await _driver.tap(incrementFloatingButton);
+    expect(await _getTextByScreen(counterState), '2');
+  }
+
+  Future<void> decrementFloatingActionButton() async {
+    await _driver.tap(decrementFloatingButton);
+    expect(await _getTextByScreen(counterState), '1');
+  }
+
+  Future<String> _getTextByScreen(SerializableFinder finder) async {
+    return await _driver.getText(finder);
+  }
```

Comment:
Thoughts on removing this method and just using `_driver.getText` directly? I think in this case the helper function isn't doing much and is actually making it more difficult to understand the test because there is an additional layer of indirection. Thoughts?

Reply:
yes you are right @felangel 
let me change it
Thank

Reply:
Done @felangel